FreeRTOS/Test/VeriFast/include/proof/list.h

```diff
@@ -50,327 +52,325 @@ typedef ssize_t   BaseType_t;
 #define listSET_SECOND_LIST_ITEM_INTEGRITY_CHECK_VALUE( pxListItem )
 
 /* Max value stored in sentinel xListEnd element */
-#define portMAX_DELAY    UINT_MAX
+#define portMAX_DELAY UINT_MAX
 
 struct xLIST;
 
-struct xLIST_ITEM
-{
-    TickType_t xItemValue;
-    struct xLIST_ITEM * pxNext;
-    struct xLIST_ITEM * pxPrevious;
-    void * pvOwner;
-    struct xLIST * pxContainer;
+struct xLIST_ITEM {
+	TickType_t xItemValue;
+	struct xLIST_ITEM * pxNext;
+	struct xLIST_ITEM * pxPrevious;
+	void * pvOwner;
+	struct xLIST *pxContainer;
 };
 typedef struct xLIST_ITEM ListItem_t;
 
-typedef struct xLIST
-{
-    UBaseType_t uxNumberOfItems;
-    struct xLIST_ITEM * pxIndex;
-    #ifdef VERIFAST /*< ***change MiniList_t to ListItem_t*** */
-        struct xLIST_ITEM xListEnd;
-    #else
-        MiniListItem_t xListEnd;
-    #endif
+typedef struct xLIST {
```

Comment:
Is there a reason this is using different brace style than our uncrustify config?

Reply:
I can fix these, but note that we're not running uncrustify on these files as it would break the special VeriFast comments. So there might be other errors to